[2025-03-17T11:07:21.271+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 7_house_assignment.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2025-03-17T11:07:21.287+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 7_house_assignment.transform_data scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2025-03-17T11:07:21.288+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-03-17T11:07:21.308+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_data> on 2024-01-01 00:00:00+00:00
[2025-03-17T11:07:21.317+0000] {standard_task_runner.py:60} INFO - Started process 1133 to run task
[2025-03-17T11:07:21.321+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', '7_house_assignment', 'transform_data', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/7_1_house_assignment.py', '--cfg-path', '/tmp/tmpg0gfm_s0']
[2025-03-17T11:07:21.323+0000] {standard_task_runner.py:88} INFO - Job 58: Subtask transform_data
[2025-03-17T11:07:21.406+0000] {task_command.py:423} INFO - Running <TaskInstance: 7_house_assignment.transform_data scheduled__2024-01-01T00:00:00+00:00 [running]> on host 13d00b3af9e0
[2025-03-17T11:07:21.586+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='7_house_assignment' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2025-03-17T11:07:21.616+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-03-17T11:07:21.626+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=7_house_assignment, task_id=transform_data, execution_date=20240101T000000, start_date=20250317T110721, end_date=20250317T110721
[2025-03-17T11:07:21.895+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-17T11:07:21.923+0000] {taskinstance.py:3280} INFO - 2 downstream tasks scheduled from follow-on schedule check
